cg/services/order_validation_service/workflows/fastq/models/sample.py

```diff
@@ -0,0 +1,14 @@
+from cg.models.orders.sample_base import PriorityEnum, SexEnum
+from cg.services.order_validation_service.constants import ElutionBuffer
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class FastqSample(Sample):
+    concentration_ng_ul: float | None = None
+    elution_buffer: ElutionBuffer
+    priority: PriorityEnum
+    quantity: int | None = None
+    require_qc_ok: bool
+    sex: SexEnum
+    source: str  # Required if no internal id
+    tumour: bool = False
```

Comment:
@islean Should I override the `enumerated_samples` property with the correct return type as it is done in Microsalt (but not in Mutant 👀 )? I considered it unnecessary but maybe there is some functionality I didn't consider

Reply:
Humm, good question. I think we override it if it makes the type hints and attributes work better, which we will see when we add rules for the workflow. My guess is that we have some rule which is only applicable for Microsalt (out of the as of yet implemented order types either way), so when accessing attributes it only made sense if the .samples was certain to return a list of MicrosaltSamples.

Reply:
add overriding of enumerated_samples

Reply:
add for mutant too